de/django_forms/README.md

```diff
@@ -436,13 +436,13 @@ Mal sehen, ob das alles auch auf PythonAnywhere funktioniert. Zeit für ein weit
 
 {% filename %}PythonAnywhere command-line{% endfilename %}
 
-    $ cd ~/<your-pythonanywhere-username>.pythonanywhere.com
+    $ cd ~/<deine pythonanywere-Domain>.pythonanywhere.com
     $ git pull
     [...]
     
 
-(Denke daran, `<your-pythonanywhere-username>` durch deinen PythonAnywhere-Benutzernamen zu ersetzen - ohne spitze Klammern).
+(Denk daran, `<deine pythonanywere-Domain>` durch deine tatsächliche PythonAnywhere-Subdomain zu ersetzen - ohne die spitzen Klammern.)
```

Comment:
"ohne spitze Klammern" was fine. Will change back to that on Crowdin.

(With the definite article, i.e., "ohne die spitzen Klammern" is correct, too, but unnecessary.)

Reply:
On second though, no, I'll leave it with the definite article, as that's close to the current English original which also uses the definite article here.

Reply:
pythonanywere --> pythonanywhere

PythonAnywhere-Subdomain <-> pyhtonanywhere-Domain